aggregator/counter_elem.gen.go

```diff
@@ -27,6 +27,8 @@ package aggregator
 import (
 	"fmt"
 
+	"math"
+
```

Comment:
newlines?

Reply:
Yeah unfortunately these are auto-generated by genny. I can take a look to see if it's easy to clean up the imports when I have some down time.